app/controllers/passwords_controller.rb

```diff
@@ -0,0 +1,42 @@
+class PasswordsController < ApplicationController
+  before_action :redirect_users_to_home
```

Comment:
heh yes, that's not very clear. 

I'm having a hard time getting a good descriptive name for this "protection";
 - `:require_anonymous_user`  ?
 - `:require_visitor` ?
 - `:redirect_authenticated_users` ?
 - `:disallow_authenticated_users` ?

What would you suggest? For now I'm going with the latter, `:disallow_authenticated_users`.